examples/remix/remix.config.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @type {import('@remix-run/dev/config').AppConfig}
+ */
+module.exports = {
+  appDirectory: "app",
+  browserBuildDirectory: "public/build",
+  publicPath: "/build/",
```

Comment:
Seems like this is configurable so it should probably read the config file here to determine the default routes like we do for angular and some other frameworks

Reply:
That's a very good point! Agreed.

I'd like to open a separate PR for that, though, because there are currently a lot of people deploying Remix (it just went GA earlier this week), and it doesn't work well on Vercel.

It's probably gonna take some time to get that config reading logic to be reliable. I'll also have to generate the routes, not just the output directory value.